lib/utils/analytics.js

```diff
@@ -63,17 +81,9 @@ class AnalyticsCollector {
     // update client_id
     this.parameters['client_id'] = this.settings.usage_analytics.client_id;
 
-    // update queueLength
     if (this.settings.usage_analytics.enabled) {
-      try {
-        const stat = fs.statSync(this.__getLogFileLocation());
-        this.queueLength = stat.size / 400; // the actual value doesn't matter
-      } catch (err) {
-        // ignore
-      }
+      this.initialize();
```

Comment:
shouldn't we use `await` over here as it is async method

Reply:
This method is not called form a async context hence can't put async here. 
The real question is do I even need to call this from here? Because I am already awaiting for initialize where it is needed. My reasoning was that this would make things faster, but I think that is an incorrect assumption. Will re evaluate once and update.

Reply:
removed the `initialize` call